### ISSUE TITLE
Documentation Pages Missing

### ISSUE THREAD

**Issue by slnickerson** [spacetelescope/jdaviz#1781]
Hello!

I'm not sure if this is the correct place to bring this up, but the documentation for Jdaviz does not include a section of who/where to ask for assistance (https://jdaviz.readthedocs.io/en/latest/index.html).

I'm trying to get Jdaviz running and I'm fairly certain instructions are missing on the documentation. There are the installation instructions, and I went through the conda route. However after that there are no instructions on how to launch the GUI or how to get the jdaviz command working on the command line (it does not for me). 

On GitHub (https://github.com/spacetelescope/jdaviz) there is a small quick start section, that links to a quick start page that does not exist (https://jdaviz.readthedocs.io/en/latest/quickstart.html). 

So in summary, it looks like the documentation needs:
1) A quick start page to help people get it up and running
2) Somewhere to ask for help. Because I cannot get it running but I do not know who to ask.

Thank you very much and I look forwards to using jdaviz!

**Comment by pllim** [comment]
Thanks. This will be fixed by #1768
